image.yaml

```diff
@@ -76,7 +74,9 @@ modules:
           - name: os-logging
 packages:
       repositories:
-          - jboss-os
+          - name: jboss-os
+            url:
+               repository: http://git.app.eng.bos.redhat.com/git/jboss-container-tools.git/plain/repos/jboss-rhel-os.repo
```

Comment:
@gaohoward beware, that it's internal URL that shouldn't be exposed/leaked in public repository.

Reply:
thanks. It's removed.